README.md

```diff
@@ -1,2 +1,100 @@
 # Awesome_Books
 Creation Book List Library for a user to store the list of books 
+
+> A demo website for Book Library.
+
+## Built With
+
+HTML and CSS
+Javascript : Arrays and Objects , DOM , Local Storage etc 
+
+## Video
+
+In the following [link]()
+
+
+## Live Server
+
+In the following link, you can view ()).
+
+## Screenshot
+
+![screenshot_TechnoCamp]()
+
```

Comment:
- [ ] **Required:** Let's ensure the screenshot is working. This gives the user a visual sample of the app.